app/controllers/articles_controller.rb

```diff
@@ -23,7 +23,7 @@ def create
     if @article.save
       redirect_to @article
     else
-      render 'new'
+      render "new"
```

Comment:
đối với chuỗi thông thường thì dùng `'`, chuỗi có giá trị của biến hoặc phép tính `"`
ex: "hello #{world}"

Reply:
@khanh227 I'm clearly and I have fixed it